### PR TITLE
Add makefile target to install knativekafka with keda

### PR DESCRIPTION
**Pull request by Cali0707**
Part of enabling tests to run with KEDA for eventing-kafka-broker.

/cc @pierDipi

### CODE REVIEW

**Comment by pierDipi** [comment]
@Cali0707 
```
./hack/lib/serverless.bash: line 409: ENABLE_KEDA: unbound variable
```

**Review by pierDipi — COMMENTED** [review]
Default value can be provided in https://github.com/openshift-knative/serverless-operator/blob/main/hack/lib/vars.bash

File: hack/lib/serverless.bash [hunk 1]
```diff
@@ -406,6 +406,10 @@ EOF
     yq write --inplace "$knativekafka_cr" spec.high-availability.replicas 1
   fi
 
+  if [[ $ENABLE_KEDA == "true" ]]; then
+    yq write --inplace "$knativekafka_cr" 'spec.config-kafka-features."controller.autoscaler"' "enabled"
```
Comment (pierDipi) [inline hack/lib/serverless.bash:410]: note: make sure that this is compatible with yq 3.4.*

Comment (Cali0707) [reply hack/lib/serverless.bash:410]: this was tested locally with:
```
yq -V     
yq version 3.4.1
```
would that work @pierDipi ?

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**